Cargo.toml

```diff
@@ -71,7 +70,7 @@ fvm_ipld_blockstore = "0.2"
 fvm_ipld_car = "0.7"
 fvm_ipld_encoding = "0.4"
 fvm_ipld_hamt = "0.7"
-fvm_shared2 = { package = "fvm_shared", version = "~2.5", default-features = false, features = ["testing"] }
+fvm_shared2 = { package = "fvm_shared", version = "~2.5", default-features = false, features = [] }
```

Comment:
How do you feel about removing the `default-features` and `features` keys? There are no default features, and there's no point having an empty features array